auto_submit/lib/requests/check_pr_handler.dart

```diff
@@ -0,0 +1,178 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
```

Comment:
name the file in accordance with `class` name: check_pr_handler => check_pull_request.

Reply:
Done, thanks keyong.